### PR TITLE
Added: abs: tensorflow_frontend

### LINKED ISSUE #10680: abs

### PR DESCRIPTION
**Pull request by Pratit23**
Close #10680 

Added abs in math.py with following tasks:
- test for abs in test_math.py
- function for abs

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/math.py [hunk 1]
```diff
@@ -424,6 +424,12 @@ def rsqrt(x, name=None):
 def nextafter(x1, x2, name=None):
     return ivy.nextafter(x1, x2)
 
+@to_ivy_arrays_and_back
+def nextafter(x1, x2, name=None):
+    return ivy.nextafter(x1, x2)
+
```
Comment (fspyridakos) [inline ivy/functional/frontends/tensorflow/math.py:430]: Please remove the duplicate nextafter

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 2]
```diff
@@ -1469,3 +1469,30 @@ def test_tensorflow_log_softmax(
...
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-02,
+        x=x[0],
+    )
```
Comment (fspyridakos) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:1498]: end new line at EOF

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 3]
```diff
@@ -1469,3 +1469,30 @@ def test_tensorflow_log_softmax(
         on_device=on_device,
         logits=x[0],
     )
+
+# abs
+@handle_frontend_test(
+    fn_tree="tensorflow.math.abs",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (fspyridakos) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:1477]: Can you make this `"numeric"`

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just the few comments, there are some lint errors, can you please follow the guide for adding a pre-commit hook which can fix them for you.

**Comment by Pratit23** [comment]
Made the required changes

File: ivy/functional/frontends/tensorflow/math.py [hunk 4]
```diff
@@ -425,6 +425,10 @@ def nextafter(x1, x2, name=None):
     return ivy.nextafter(x1, x2)
 
 
+def abs(x, name=None):
```
Comment (fspyridakos) [inline ivy/functional/frontends/tensorflow/math.py:428]: Please add the unit dtypes as unsupported dtypes.

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Thanks for the changes, just one last thing and you're golden

File: ivy/functional/frontends/tensorflow/math.py [hunk 5]
```diff
@@ -425,6 +425,14 @@ def nextafter(x1, x2, name=None):
     return ivy.nextafter(x1, x2)
 
 
+@with_unsupported_dtypes(
+    {"1.2.0": ("float16", "complex64", "complex128"), "1.8.0 and below": ("float16")},
+    "tensorflow",
+)
```
Comment (fspyridakos) [inline ivy/functional/frontends/tensorflow/math.py:431]: Please include the uints as unsupported from 2.9.0 and below  in order to pass the test (i.e. `uint8`, `uint16`, ...)

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Still have some unsupported dtypes to add, also please commit using the pre-commit hook as described here: https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit
This is to avoid formatting errors

File: ivy/functional/frontends/tensorflow/math.py (hunk 5 above)
Comment (Pratit23) [reply ivy/functional/frontends/tensorflow/math.py:431]: This commit was passed through the pre-commit hook

**Comment by Pratit23** [comment]
Passed the latest commit through the pre-commit hook and added the missing dtypes

**Comment by fspyridakos** [comment]
LGTM, thanks for your contribution again!

**Pull request merged**